tools/gen_openvino_dockerfile.py

```diff
@@ -123,37 +123,19 @@ def dockerfile_for_windows(output_file):
     df += """
 SHELL ["cmd", "/S", "/C"]
 
-# Build instructions:
-# https://github.com/openvinotoolkit/openvino/wiki/BuildingForWindows
+# instructions:
```

Comment:
Can you make a note that we are building from source for linux but installing from the openvino repository for windows? @nnshah1 I would like your opinion on whether there should be a follow up ticket to revisit this installation fracture?

Reply:
Sure, added [Note linux is building from source while windows is using archive](https://github.com/triton-inference-server/openvino_backend/pull/63/commits/aa40ca1fd6b28ff2f0dad0dba5ffb96f5269b84b)